docs/user-guide/docs-source/appendix-contributing.md

```diff
@@ -0,0 +1,98 @@
+
+Clone this repository into the "Plugins" folder of the UE project you created. Note, you will need to create the "Plugins" folder first.
+
+`<your sample project>\Plugins\aws-ambit-scenario-designer-ue4\`
+
+Once the Ambit plugin repository has been cloned, right-click on the `*.uproject` file at the root of the project directory and select the "Generate Visual Studio project files" option. Select the appropriate engine version if required. A new Visual Studio solution file (`*.sln`) will be generated for you.
+
+Launch your sample project by clicking on the `*.uproject` file. When prompted to rebuild missing modules, click "Yes".
+
+![Missing Modules prompt](images/appendix/RebuildMissingModules.png)
+
+Once the project opens, enable the "Functional Testing Editor" plugin, and restart (or simply close) the editor for the plugin to take effect. This plugin is used to run Ambit's automated tests.
+
+
+
+## Development Workflow
+
+Follow this workflow when making changes to the Ambit C++ code...
+
+1. Launch the project's `*.sln` file in Visual Studio. 
+2. In the **Solution Explorer** window, right-click on your project and select **Set as Startup Project**. 
+3. Make your code edits.
+4. Compile and run the solution through either of these methods:
+   1. To build only: Select `Build > Build Solution` from the Visual Studio menubar
+   2. To build and run: Press the **Local Windows Debugger** button (or press **F5**).
+5. Repeat steps 3-4 as needed.
+
+
+
+## Testing
+
+### Running Automated Tests
+
+The Ambit plugin includes a number of automated tests. Automated test files, called "specs", end with a `*.spec.cpp` file extension and are usually located next to the files that they test. These tests will be automatically recognized by the Unreal Engine test runner. To run the tests:
+
+1. First open the **Test Automation** window (`Window > Test Automation`).
+2. Select the **Automation** tab.
+3. Tick the box next to the **Ambit** group of tests.
+4. Click the **Start Tests** button.
+
+> 💡 **Tip:** For more info on working with automated tests in UE see ["Automation Spec Overview"](https://docs.unrealengine.com/en-US/Programming/Automation/AutomationSpec/index.html)
+
+### Test Maps
+
+A sample map is provided to aid in manual testing of the various Ambit spawner actors. You'll find that map at:
+
+ `/Ambit Content/Test/Maps/ProceduralPlacementTestMap`
+
+### Testing Scenario Import/Export
+
+To manually test scenario import and export functionality, follow these steps:
+
+1. First set some random parameters in the **Scenario Settings** section of the Ambit Mode.
+2. Click the "Export Scenario" Button at the bottom of the **Scenario Settings** section and save the `*.sdf.json` file.
+3. Verify the file was created in the destination location you specified.
+4. Reset all the values of your **Scenario Settings** section or re-load the project.
+5. Click the '...' next to **Scenario Name** and import the `*.sdf.json` file.
```

Comment:
minor: change '...' to `...` because the monospace format for '...' been used elsewhere as well, maintains uniformity